src/languages/nsis.js

```diff
@@ -159,7 +159,7 @@ export default function(hljs) {
   const VARIABLES = {
     // $variables
     className: 'variable',
-    begin: /\$+\w+/,
+    begin: /\$+[\w\.]+/,
```

Comment:
Can variables BEGIN with a `.`?  If not we probably need `\w[\w.]+` no?

Reply:
Did a quick check and you are right, it's not allowed